src/critics/local_goal_critic.cpp

```diff
@@ -0,0 +1,109 @@
+// Copyright 2022 @artofnothingness Alexey Budyakov, Samsung Research
+#include "mppic/critics/local_goal_critic.hpp"
+
+#include <xtensor/xmath.hpp>
```

Comment:
This file needs inline comments to understand what you're doing in different phases of `score`

Reply:
added